engine/util/src/main/scala/pl/touk/nussknacker/engine/util/namespaces/NamingContext.scala

```diff
@@ -0,0 +1,11 @@
+package pl.touk.nussknacker.engine.util.namespaces
+
+// This class can be extended in the future to provide more
+// information about context (e.g. process name, category)
+class NamingContext(usageKey: ObjectNamingUsageKey.UsageKey)
```

Comment:
will it be possible to add context without changing this file? Probably not? Can we make it possible?